javascript/javascript3/week1/js3wk1-homework/weather-App/weatherApp.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JS 3 WK1 Homework</title>
+    <link rel="stylesheet" href="weatherApp.css">
+</head>
+<body>
+
+    <h1>Weather App</h1>
+
+        <div>
+            <label for="cityInput">City</label>
+            <input id="cityInput" type="text" placeholder="please enter city" required>
+            <button id="submitButton">Submit</button>    
+        </div>
+        <div>
+            <h1 class="cityNameOutput"></h1>
+            <h2 class="temperatureOutput"></h2>    
+            <img src="" class="weatherIconOutput"/>
```

Comment:
As you put `src =''`, the app got an image placeholder. You don't need it if the city is not yet typed in.